backend/layers/common/entities.py

```diff
@@ -66,6 +66,18 @@ class DatasetConversionStatus(DatasetStatusGeneric, Enum):
     SKIPPED = "SKIPPED"
 
 
+class DatasetVisibility(Enum):
```

Comment:
I'm thinking we refactor CollectionVisibility into just a Visibility enum and reuse that, rather than create a second, identical DatasetVisibility

Reply:
Agreed